src/GraphicsDeviceManager.cs

```diff
@@ -87,17 +87,26 @@ public bool SynchronizeWithVerticalRetrace
 
 		public DisplayOrientation SupportedOrientations
 		{
+			/* XNA on Windows Phone had the ability to change
+			 * the list of supported device orientations at runtime.
+			 * Unfortunately, we can't support that reliably across
+			 * multiple mobile platforms. Therefore this property is
+			 * essentially a no-op.
+			 * 
+			 * Instead, you should set your supported orientations in
+			 * Info.plist (iOS) or AndroidManifest.xml (Android).
+			 * 
+			 * -caleb
+			 */
+
 			get
 			{
 				return supportedOrientations;
 			}
 			set
 			{
+				FNALoggerEXT.LogWarn("Setting SupportedOrientations has no effect!");
 				supportedOrientations = value;
-				if (game.Window != null)
```

Comment:
... so this file can go back to the old way, and the warning can go in the FNAWindow function.

Reply:
should I keep the comment where it is or move it to FNAWindow too?

Reply:
Let's move it to FNAWindow - the GraphicsDeviceManager is doing the right thing, it's the FNAPlatform that's at fault in this case.